test/src/parallel_qp_solve.cpp

```diff
@@ -189,8 +245,8 @@ DOCTEST_TEST_CASE("test sparse vector QP")
   }
 
   const size_t NUM_THREADS = (size_t)omp_get_max_threads();
-  proxsuite::proxqp::parallel::qp_solve_in_parallel((size_t)(NUM_THREADS / 2),
-                                                    qps_vector);
+  proxsuite::proxqp::parallel::qp_solve_in_parallel(qps_vector,
```

Comment:
Why do you have two namings different between cpp and python? (`qp_solve_in_parallel` and `solve_in_parallel`)

Reply:
I don't think the namespace parallel is helpful. I would remove it. In addition, the Python bindings have an extra dense namespace. Why?

Reply:
I renamed both to `solve_in_parallel` and made in consistent between cpp and python.